### ISSUE TITLE
FeatureRequest: Prevent accidental deletion of context when swapping the new and old context in kubectx

### ISSUE THREAD

**Issue by RonaldFindling** [ahmetb/kubectx#136]
To **prevent accidental deletion of a context** when swapping the old and new context parameters on a rename operation check at first if the `oldContext` exists.

Use Case:
```
# correct usage
kubectx newContext=oldContext
# wrong usage which deletes oldContext
kubectx oldContext=newContext
```

Currently in case of wrong usage:
```
user:~$ kubectx oldContext=newContext
Context "oldContext" exists, deleting...
error: cannot rename the context "nexContext", it's not in /Users/user/.kube/config
```

Preferred in case of wrong usage: (no context gets deleted)
```
user:~$ kubectx oldContext=newContext
error: The context "nexContext", is not in /Users/user/.kube/config
```

**Comment by ahmetb** [comment]
Currently we do:

1. does the $newContext exist? if so, "delete" it first (so that rename doesn't fail)
2. kubectl config rename-context [...]

Sounds like we should instead do:

1. does the $oldContext exist, if not, QUIT with error.
2. does the $newContext exist? if so "delete" it first  (so that rename doesn't fail)
3. kubectl config rename-context [...]

Correct?

**Comment by RonaldFindling** [comment]
Exactly.

**Comment by jacobtomlinson** [comment]
Just fell into this trap :(

**Comment by ahmetb** [comment]
I prepared a fix in #139. Feel free to test it, but seems trivial.
